linode_api4/objects/networking.py

```diff
@@ -106,6 +107,32 @@ def to(self, linode):
         return {"address": self.address, "linode_id": linode.id}
 
 
+@dataclass
+class VPCIPAddress(JSONObject):
+    """
+    VPCIPAddress represents the IP address of a VPC.
+
+    NOTE: This is not implemented as a typical API object (Base) because VPC IPs
+    cannot be refreshed through the /networking/ips/{address} endpoint.
+    """
+
+    address: str = ""
+    gateway: str = ""
+    region: str = ""
+    subnet_mask: str = ""
+    vpc_id: int = 0
+    subnet_id: int = 0
+    linode_id: int = 0
+    config_id: int = 0
+    interface_id: int = 0
+    prefix: int = 0
+
+    active: bool = False
+
+    address_range: Optional[str] = None
+    nat_1_1: Optional[str] = None
+
+
```

Comment:
Do we think it'd make sense to implement property methods here to allow for easy access to the corresponding VPC, VPCSubnet, etc. objects from this class? The only issue is that a client isn't embedded in JSON objects as of now, although it would be a relatively straightforward change to make.